pkg/graph/schema.graphql

```diff
@@ -1487,6 +1515,7 @@ type Query {
   possibleOperationalNeeds: [PossibleOperationalNeed!]!
   possibleOperationalSolutions: [PossibleOperationalSolution!]!
   userAccount(username: String!): UserAccount!
+  operationalSolutionSubtask(id: UUID!): OperationalSolutionSubtask

```

Comment:
I'm not sure this needs to be exposed. Was there something in FIGMA that suggests the front end would ever need to get a specific SubTask by ID? I could see the front end getting the collection by solution ID however. I think we mostly would need get by id for pre-update statements.

Reply:
We had briefly discussed it in our sync around operational solution subtasks. My understanding was that we intended to provide it as a matter of good practice / API completion. If you prefer to remove the endpoint it's certainly quick and easy to do, but it seems to me that we only lose platform capability.

Reply:
hmm 🤔. I think the main thing I think of for this is if we aren't going to need it, we probably shouldn't have it. I know they AC did bring up collection by solution ID, I just don't know that this is needed. 

@patrickseguraoddball, do you mind weighing if about querying a singular solution subtask on the front end? Also, do anticipate a need to just get the subtasks by solution id? Or is nested in the solution sufficient?

Reply:
I just looked through the Figma, and I don;t see any instance we we would query a singular subtask or query subtasks without needed the full need/solution.  I can see a future scenario where this may change, but I don't believe currently needed

Reply:
Thanks so much @patrickseguraoddball! I think in that case @OddTomBrooks , I'd remove it so we don't need to support something we don't anticipate needing.

Reply:
Makes sense. The underlying capability is still there for internal purposes, so it's trivial if we ever need to restore it.